docker/php-fpm/src/config.php

```diff
@@ -63,28 +63,30 @@
 ];
 
 // Define other looking glasses, this is useful if you have multiple networks and looking glasses;
-const LG_LOCATIONS = [
+/* const LG_LOCATIONS = [
     'Location A' => 'https://github.com/hybula/lookingglass/',
```

Comment:
Why remove the examples? Easier to keep them commented to show people how it works. Remember that most people installing such looking glass aren't that technical in terms of coding 😄

Reply:
Sorry I thought changing the default value to empty would allow the location drop-down menu to auto hide out of the box. I did keep them in the comments though. Should I change these back?